react/components/QuantityStepper.tsx

```diff
@@ -139,12 +139,15 @@ const QuantityStepper: VFC<Props> = ({
     <div className="flex">
       <button
         className={classnames(
+          styles.quantityButton,
+          styles.quantityButtonDecrease,
           'pa4 ba br2 br--left flex items-center justify-center',
           {
             'c-muted-1 b--muted-4 hover-b--muted-3 bg-muted-5 hover-bg-muted-4 pointer': !disabled,
             'bg-muted-5 c-muted-3 b--muted-4': disabled,
           }
         )}
+        data-quantity-action="decrease"
```

Comment:
```suggestion
```

Reply:
If this suggestion is applied, remember to remove on changelog